### DISCUSSION TITLE
point_0714

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#269] [Roy]
"20190622T180929_20190622T181355_T12TVK"
![image](https://user-images.githubusercontent.com/54947195/147610253-6bbac2b2-2a1f-4c11-af5e-a03bb83e6cbc.png)

"20200316T182101_20200316T182943_T12TVL"
![image](https://user-images.githubusercontent.com/54947195/147610352-7b20384a-a976-4ecb-abdf-e119069fad7e.png)

"20200604T181931_20200604T182227_T12TVL"
![image](https://user-images.githubusercontent.com/54947195/147610453-9a1d877d-4b95-440f-bd1f-f7eb59155dcc.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20190622T180929_20190622T181355_T12TVK"
![Captura](https://user-images.githubusercontent.com/86502862/153492466-0d57fef6-17db-4f92-9c6a-bedcab1d5a78.JPG)

**Comment by AntoniCesar** [comment]
"20200604T181931_20200604T182227_T12TVL"
![Captura](https://user-images.githubusercontent.com/86502862/153503103-43e5c877-2740-4b7f-b395-33e1c6682719.JPG)
